packages/ckeditor5-dev-env/lib/release-tools/tasks/update-package-versions.js

```diff
@@ -0,0 +1,150 @@
+/**
+ * @license Copyright (c) 2003-2021, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const chalk = require( 'chalk' );
+const { diffChars: diff } = require( 'diff' );
+const { execSync } = require( 'child_process' );
+const fs = require( 'fs' );
+const glob = require( 'glob' );
+
+/**
+ * Updates `@ckeditor/ckeditor5-*` and `ckeditor5` dependencies in `packages/*` and `release/*`
+ * directories to the latest version. Changes in `packages/*` will be committed as well.
+ *
+ * See https://github.com/cksource/ckeditor5-internal/issues/1123
+ *
+ * @param {Object} options
+ * @param {String} options.cwd The root directory of the repository in which
+ * the script is being used.
+ * @param {Boolean} options.dryRun Prevents the script from committing, and
+ * instead shows list of files that was changed in `packages/*` directory.
+ * @param {Boolean} options.pathsToUpdate Instead of list of files, it displays detailed
+ * list of changes from each file. Has no effect without options.dryRun.
+ */
+module.exports = function updatePackageVersions( options ) {
+	const totalResult = { found: 0, updated: 0, differences: [] };
+	const pathsToCommit = [];
+
+	options.pathsToUpdate.forEach( ( value, index, array ) => {
+		array[ index ].path = value.path.split( '\u005C' ).join( '\u002F' );
```

Comment:
What is this magic? Consider whether it can not be done more simply, without using Unicode escape sequence `\uXXXX` and never modify the original input argument.